lib/processor.js

```diff
@@ -5,6 +5,11 @@
  * @returns object { mergeable: bool, description: string | [string]}
  */
 const processFilters = (validatorContext, input, filters) => {
+  // if input is null or undefined, turn it into empty string
+  if (!input) {
```

Comment:
To be on the safe side I propose using a double bang here. i.e. `(!!input)`

Reply:
standard said `no double negation`, would you prefer i don't fix this ? or fix standard?